core/extension/openseadragon-canvas-draw-overlay.js

```diff
@@ -1023,6 +1024,11 @@
       if (this.drawMode == 'point') {
         this._current_path_ = {
           type: 'Feature',
+          viewerStates: {
+            x: StatesHelper.getCurrentStates(true).x,
+            y: StatesHelper.getCurrentStates(true).y,
+            z: StatesHelper.getCurrentStates(true).y
```

Comment:
```suggestion
            z: StatesHelper.getCurrentStates(true).z
```
I'm pretty sure this is why this is happening:
![Screenshot 2024-07-19 at 10 49 08 AM](https://github.com/user-attachments/assets/0267d02e-9080-4385-8a77-80e170a9dc07)
![Screenshot 2024-07-19 at 10 49 02 AM](https://github.com/user-attachments/assets/c363255a-51bb-4812-830b-aaa2f1335f6e)

Reply:
I also noticed a mistake in my local and was fixing the same area.
Thank you!!